[docs] update eas.json docs: add scheme to ios build profiles (#9933)
expo/expo: docs/pages/build/eas-json.md

{
  "workflow": "generic",
  "credentialsSource": "local" | "remote" | "auto", // default: "auto"
  "scheme": string,
  "artifactPath": string // default: "ios/build/App.ipa"
}
```

- `"workflow": "generic"` indicates that your project is a generic one.
- `credentialsSource` defines the source of credentials for this build profile. If you want to take advantage of your own `credentials.json` file, set this to `local` ([learn more on this here](../advanced-credentials-configuration/)). If you want to use the credentials Expo already has stored for you, choose `remote`. If you're not sure what to do but you probably won't be running builds from a CI, choose `auto` (this is the default option).

> - `scheme` defines the Xcode project's schema to build. You should set it if your project has multiple schemes. Please note that if the project has only one scheme, it'll be automatically detected. However, if multiple schemes exist but this value is not set, Expo CLI will prompt you to select one of them.

Reviewer: ```suggestion
- `scheme` defines the Xcode project's scheme to build. You should set it if your project has multiple schemes. Please note that if the project has only one scheme, it will automatically be detected. However, if multiple schemes exist and this value is **not** set, Expo CLI will prompt you to select one of them.
```
